kpi/tests/test_import_tasks.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+from datetime import timedelta
+
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from kpi.maintenance_tasks import remove_old_import_tasks
+from kpi.models import ImportTask
+from kpi.tests.base_test_case import BaseTestCase
+
+
+class AssetImportTaskHousekeepingTest(BaseTestCase):
+    fixtures = ['test_data']
+
+    def setUp(self):
+        self.user = User.objects.get(username='someuser')
+
+    def test_remove_old_import_tasks(self):
+        old_task = ImportTask.objects.create(
+            user=self.user,
+            data='{}',
+        )
+
+        old_task.date_created = timezone.now() - timedelta(days=95)
```

Comment:
I assume we couldn't set date_created directly and this works around it. Does it work when using model bakery?

Reply:
Correct, we can't set it on creation. Doesn't work with model bakery, either. I have added a note